### DISCUSSION TITLE
Command line switches for Windows installation

### DISCUSSION THREAD

**Discussion by elementaltoad** [LizardByte/community#224] [Sunshine] [answered]
I'd like to be able to deploy Sunshine to a few remote machines of mine, are there currently any command line switches for silent or unattended installations in Windows environments? Thanks!

**Comment by ReenigneArcher** [comment, marked as answer]
`sunshine-windows.exe /S`

Can change directory with the `/D` option. e.x `/D=C:\Program Files\Sunshine`
